components/editor/modules/list/index.js

```diff
@@ -119,9 +123,8 @@ export default ({rule, subModules, TYPE}) => {
           return change.splitBlock(2)
         },
         schema: {
-          rules: [
-            {
-              match: matchBlock(TYPE),
+          blocks: {
+            [TYPE]: {
               validate: node => {
```

Comment:
is this still supported? afaik we now only have `validateNode` as a plugin fn and only a `normalize` fn for new schema rules.